testdata/interfacetotype/interfacetotype.go

```diff
@@ -0,0 +1,24 @@
+package codersdk
+
+type Player struct {
+	User
+	Score int `json:"score"`
+}
```

Comment:
Inheritance is not currently working

Reply:
Fixed with type intersections!

https://github.com/coder/guts/pull/52/commits/30fd888c5fe36a558058d4652371fe4f781c21c4